azkaban-common/src/test/java/azkaban/executor/InteractiveTestJob.java

```diff
@@ -141,4 +142,10 @@ public void cancel() throws InterruptedException {
     info("Killing job");
     failJob();
   }
+
+  public static void clearTestJobs(final String... names) {
```

Comment:
Would synchronization be needed?
Note that in the future we may want to run tests in parallel.